packages/flutter/test/widgets/focus_traversal_test.dart

```diff
@@ -3786,6 +3786,75 @@ void main() {
     expect(childScope.hasFocus, isTrue);
     expect(nodeA.hasFocus, isFalse);
   });
+
+  testWidgets('GIVEN onFocusNodeCreated is not null '
```

Comment:
Do you want to add a test for your specific use case: have it prevent the `FocusScope` from receiving focus by listening to the node creation?